test/e2e/unpack_test.go

```diff
@@ -91,9 +92,9 @@ var _ = Describe("Catalog Unpacking", func() {
 			name := strings.Split(url.Hostname(), ".")[0]
 			port := url.Port()
 			// the ProxyGet() call below needs an explicit port value, so if
-			// value from url.Port() is empty, we assume port 80.
+			// value from url.Port() is empty, we assume port 443.
 			if port == "" {
-				port = "80"
+				port = "443"
```

Comment:
I think this is the cause of the e2e failure we are seeing. Looks like by default we are expecting to use HTTPS (as such port 443) for the ProxyGet operation. This was probably hardcoded to 443 because upstream the default is to use HTTPS and that is what we are testing against. Downstream we are currently still running with HTTP because we haven't yet configured service-ca operator stuff for downstream to enable HTTPS.

My recommendation to get the e2e to pass would be to add a drop commit that sets the port variable based on the `url.Scheme` value:
- HTTP --> 80
- HTTPS --> 443

We could then fix this in the same way upstream for the next sync.

Reply:
It would be better to fix this upstream, and then pull...

Reply:
I've added a drop commit.

Reply:
And it still fails... :/